### ISSUE TITLE
boundingBox 'altitude' rule change

### ISSUE THREAD

**Issue by stansmith907** [adiwg/mdJson-schemas#154]
Schema rules for *geographicExtent.boundingBox.altitude*:

Definition *altitude* "The limits of coverage of a data set expressed by altitude. The term 'altitude' is used instead of the common term 'elevation' to conform to the terminology in Federal Information Processing Standards 70-1 and 173.  Height above sea level.

Definition *minimumAltitude*: "The minimum altitude extent of coverage."

Definition *maximumAltitude*: "The maximum altitude extent of coverage."

Definition *unitsOfAltitude*: "The units of measure in which altitude is expressed."  Definition is missing in the FGDC documentation, so I made it up.

 - [x] mininimumAltitude real
 - [x] mininimumAltitude optional
 - [x] maxinimumAltitude real
 - [x] maxinimumAltitude optional
 - [x] unitsOfAltitude string
 - [x] unitsOfAltitude required if either minimum or maximum are present

Other considerations:
 - [x] spatial bounds also required if altitude bounding present
 - [x] altitude bounds will not be calculated for computed bounding box when coordinates have third dimension. 

mdJson:
````json
  "boundingBox": {
      "westLongitude": -140.0,
      "eastLongitude": -120.0,
      "southLatitude": 49.0,
      "northLatitude": 70.0,
      "minimumAltitude": 250.0,
      "maximumAltitude": 1500.0,
      "unitsOfAltitude": "meters"
    }
````

FGDC mapping:
````xml
<metadata>
  <idinfo>
    <spdom>
      <descgeog>spatial domain description</descgeog>
      <bounding>
        <westbc>-166.0</westbc>
        <eastbc>-74.00</eastbc>
        <northbc>71.0</northbc>
        <southbc>24.00</southbc>
        <boundalt>
          <altmin>100.0</altmin>
          <altmax>1000.0</altmax>
          <altunit>meters</altunit>
        </boundalt>
      </bounding>
    </spdom>
  </idinfo>
</metadata>
````

ISO 19115-2 mapping: none

**Comment by jlblcc** [comment]
Note: the constraint "spatial bounding box also required if altitude bounding present" applies. That's because all of the bounding coordinates are required in a boundingBox instance. This would need to change if we want to allow altitude bounds only.

**Comment by stansmith907** [comment]
Yeah, put is that practical?  What would an elevation, altitude, or depth be without a location on the earth?  A satellite orbit?

**Comment by stansmith907** [comment]
Apparently, point missed  :wink:
